docs/tracker.rst

```diff
@@ -0,0 +1,36 @@
+Tracker Reference
+^^^^^^^^^^^^^^^^^
+
+Intro
+-----
+Tracker is a Rally utility for generating a Rally track template + corpus from one or more existing Elasticsearch
+indices.
+
+Usage
+-----
+To generate a new track, invoke Tracker as follows (replacing the example parameters below)::
+
+    estracker --target-hosts=host:9200,host2:9200 --indices=index_1 index_2 --trackname=xtrack --outdir=mytracks
```

Comment:
We should also document the `--client-options` e.g. with an example; scrapping real Cloud clusters will be THE most common use case.

Reply:
Also there's a typo here it's `--track-name` not `--trackname`

Reply:
Additionally the values for `--indices` should be enclosed in double quotes when dealing with multiple indices.

Reply:
track-name and client-options: https://github.com/elastic/rally/pull/875/commits/59e9bb311c4a6b60a351cc699a156f585e46d49c

csv_to_list indices handling: https://github.com/elastic/rally/pull/875/commits/f962abf5f4b7fc84a449333e0617caa322fd7aaf